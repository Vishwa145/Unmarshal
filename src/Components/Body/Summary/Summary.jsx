import React from 'react';

function Summary (){
    return(
        <div className="Summary">
            <h3><strong>Transaction Details</strong></h3><p>Unmarshal - sponsered slots available <a href="/">Book your slot here!</a></p>
        </div>
    );
}

export default Summary;
