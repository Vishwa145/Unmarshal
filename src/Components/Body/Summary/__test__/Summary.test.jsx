import React from "react";
import ReactDOM from "react-dom";
import Summary from "../Summary";
import renderer from "react-test-renderer"

it("render successfully", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Summary />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("Matches snapshot", ()=>{
    const tree = renderer.create(<Summary />).toJSON();
    expect(tree).toMatchSnapshot();
});