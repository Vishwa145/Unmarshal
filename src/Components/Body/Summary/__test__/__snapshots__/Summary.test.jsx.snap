// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="Summary"
>
  <h3>
    <strong>
      Transaction Details
    </strong>
  </h3>
  <p>
    Unmarshal - sponsered slots available 
    <a
      href="/"
    >
      Book your slot here!
    </a>
  </p>
</div>
`;
